pkg/validation/linters_config_test.go

```diff
@@ -163,6 +163,34 @@ var _ = Describe("LintersConfig", func() {
 			expectedErr: "lintersConfig.jsonTags.jsonTagRegex: Invalid value: \"^[a-z][a-z0-9]*(?:[A-Z][a-z0-9]*\": invalid regex: error parsing regexp: missing closing ): `^[a-z][a-z0-9]*(?:[A-Z][a-z0-9]*`",
 		}),
 
+		// NoMapsConfig validation
+		Entry("With a valid NoMapsConfig", validateLintersConfigTableInput{
+			config: config.LintersConfig{
+				NoMaps: config.NoMapsConfig{
+					Policy: "",
+				},
+			},
+			expectedErr: "",
+		}),
+
+		Entry("With a valid NoMapsConfig: allowStringToStringMaps is true", validateLintersConfigTableInput{
+			config: config.LintersConfig{
+				NoMaps: config.NoMapsConfig{
+					Policy: config.NoMapsAllowStringToStringMaps,
```

Comment:
Please test with the other two valid values as well please

Reply:
Added.